plugins/outputs/kairosdb/kairosdb.go

```diff
@@ -0,0 +1,287 @@
+package kairosdb
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var (
+	allowedChars = regexp.MustCompile(`[^a-zA-Z0-9-_./\p{L}]`)
+	hypenChars   = strings.NewReplacer(
+		"@", "-",
+		"*", "-",
+		`%`, "-",
+		"#", "-",
+		"$", "-")
+	defaultHttpPath  = "/api/v1/datapoints"
+	defaultSeperator = "_"
+)
+
+type KairosDB struct {
+	Prefix string
+
+	Host string
+	Port int
+
+	Username string
+	Password string
+
+	HttpBatchSize int // deprecated httpBatchSize form in 1.8
```

Comment:
Is this still needed? Or can we use a sensible constant that works for everyone?